web/packages/core/src/ruffle-player.ts

```diff
@@ -148,6 +158,8 @@ export class RufflePlayer extends HTMLElement {
 
     private isExtension = false;
     private longPressTimer: ReturnType<typeof setTimeout> | null = null;
+    private pointerDownPosition = new Point(-1, -1);
```

Comment:
I don't know if this makes sense, but initalizing this with `null` seems more clear to me. Then a non-null assertion will be needed in `checkLongPressMovement`.

Reply:
Ok, updated

Reply:
... and resolved linter's complaint, oops